### DISCUSSION TITLE
activity type desc

### DISCUSSION THREAD

**Discussion by tianweimol** [elsa-workflows/elsa-core#1075] [Q&A]
thanks for your answer
i saw there are many activity type.how can i find out further information what descripted every activity type?
can i find the description in some document？
eg.
readline activity description:...
break activity description:...
tks

**Comment by sfmskywalker** [comment]
Right now there is no documentation yet, but I'll be working on that over the next couple of weeks and months.
Meanwhile, your best bet is to look at the activity description and take a guess and try them out. Some of them are also used in sample projects to demonstrate their use cases.

And of course, feel free to ask here on GitHub or on our Discord channel.
